episodes/lesson-content.md

```diff
@@ -419,7 +413,7 @@ Use this time to add more content to your lesson site.
 Generally speaking, you should try to transfer the drafted content from your lesson design notes into the lesson website.
 Here are some suggestions for things you might try:
 
-1. Add your list of prerequisite knowledge/skills as a fenced div with the `prereq` class to your `index.md` file.
+1. Add different types of exercises - e.g. add a `discussion`.
```

Comment:
I like this suggestion, but do we need to add a note about `solution`s not working well with `discussion`s?

Reply:
Is this the issue where "solutions" do not get rendered within "discussions" @tobyhodges ? I forgot ehat the issue was but remember that something was wrong there.

Reply:
That's right: the Workbench assumes that discussions do not have a solution.

Reply:
```suggestion
1. Add other exercises - e.g. add a `discussion`. Note that the lesson infrastructure does not support `solution` divs to be attached to `discussion`s.
```